docs/stdlib/doc.md

```diff
@@ -0,0 +1,45 @@
+## Builtin VS StdLib
+
+[Builtins](/advanced_syntax/builtins) are method that are included in the Amber compiler and don't need to be imported in the code.
+
+In contrast, the standard library (stdlib) is a collection of Amber functions that are embedded in every Amber release. Each version of Amber may include changes to the standard library and you need to import these functions in your code. These functions are more advanced and can accept various parameters.
```

Comment:
we need to explain the difference between those, we discussed internally why something should became a builtin or a stdflib so deserve a mention in the docs

Reply:
But this section doesn't explain when something should be builtin or stdlib. It just says that you don't need to import builtin and stdlib you have to import. It's basic truth and I too don't see much sense in writing this

Reply:
I think that this section is important to explain the difference between the two.
Maybe the wording is not the best one but it's a start.

If you have a suggestion on how to improve it right now I will change it as I already did since the PR started.